src/components/organisms/deployments/sessions/tabs/activities/infiniteList.tsx

```diff
@@ -24,6 +24,20 @@ export const ActivityList = () => {
 		t,
 	} = useVirtualizedList<SessionActivity>(SessionLogType.Activity, 60);
 
+	const [rowHeight, setRowHeight] = useState(60);
+
+	useEffect(() => {
+		const handleResize = () => {
+			setRowHeight(window.innerWidth < 1500 ? 80 : 60);
+		};
+
+		handleResize();
+
+		window.addEventListener("resize", handleResize);
+
+		return () => window.removeEventListener("resize", handleResize);
+	}, []);
+
 	const customRowRenderer = useCallback(
 		({ index, key, style }: ListRowProps) => (
 			<ActivityRow
```

Comment:
add inside ActivityRow to parent div this class `overflow-x-auto scrollbar` 
![image](https://github.com/user-attachments/assets/1285c5a4-3d29-4c6c-abf9-5e53dda2d3fb)

Reply:
Looking good, refactored